.github/CONTRIBUTING.md

```diff
@@ -195,6 +195,16 @@ python -m pip install ".[dev, examples]"
 python -m pip install pre-commit
 ```
 
+Additionally, for testing backward compatibility with older versions of Pytorch Lightning, you also need to download all saved version-checkpoints from the public AWS storage. Run the following script to get all saved version-checkpoints:
+
+```bash
+cd legacy
+wget https://pl-public-data.s3.amazonaws.com/legacy/checkpoints.zip
```

Comment:
might want to clarify which directory these must be downloaded to

Reply:
yep. I can update this like 
```bash
wget https://pl-public-data.s3.amazonaws.com/legacy/checkpoints.zip -P legacy/
unzip -o  legacy/checkpoints.zip -d legacy/
```